### PR TITLE
Add web links

### PR DESCRIPTION
**Pull request by railsstudent**
#111 

Add Events menu with unconference, lightning talks and global & local f/oss meeting
Please ask marketing team to review.

### CODE REVIEW

**Comment by tonyhhyip** [comment]
Request for screen cap

**Comment by railsstudent** [comment]
selection_015
selection_016
selection_017
selection_018

**Comment by railsstudent** [comment]
selection_019

File: assets/pages/events/global-local-foss.jinja [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <div class="container">
+            <h1>Global &amp; Local F/OSS Community Meeting</h1>
+        </div>
+    </div>
+</div>
+  <div class="container">
+      <div class="content">
+
+        <p>Registration of Global &amp; Local F/OSS Community Meeting is opening now. You are welcome to write down your topic and name.</p>
+        <a class="waves-effect waves-light z-depth-2 btn" href="https://goo.gl/forms/1LAG9T5VIng3w5MN2">Application of Global F/OSS Community Meeting</a>
```
Comment (tonyhhyip) [inline assets/pages/events/global-local-foss.jinja]: Change to "Apply"

**Review by tonyhhyip — CHANGES_REQUESTED** [review]
Global & Local F/OSS Community Meeting should put in two different pages

**Comment by railsstudent** [comment]
Fixed

**Comment by railsstudent** [comment]
selection_024
selection_023
selection_022
selection_021
selection_020

**Comment by tonyhhyip** [comment]
I will put it in beta for marketing team to review for the pages

**Comment by tonyhhyip** [comment]
Please rename `index.jinja` -> `unconference.jinja`

**Comment by railsstudent** [comment]
Fixed.

**Comment by railsstudent** [comment]
selection_012

File: assets/pages/events/lightning-talk.jinja [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <div class="content">
+        <div class="container">
+            <h1>Lightning Talks</h1>
+        </div>
+    </div>
+</div>
+  <div class="container">
+      <div class="content">
+
+        <p>Lightning talks will be held in Day 1 Afternoon (3:00pm - 3:25pm).</p>
```
Comment (tonyhhyip) [inline assets/pages/events/lightning-talk.jinja]: Lightning Talk is on Day 2

**Review by tonyhhyip — CHANGES_REQUESTED** [review]

**Comment by railsstudent** [comment]
selection_017

**Review by tonyhhyip — APPROVED** [review]

**Pull request merged**